test/e2e/metrics.spec.js

```diff
@@ -15,7 +15,7 @@ describe('Segment metrics', function () {
         {
           secretKey:
             '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
-          balance: 25000000000000000000,
+          balance: '0x15af1d78b58c40000',
```

Comment:
Might be nice to bring in BN and use `0x${new BN(25000000000000000000, 10).toString(16)}` (or make a function that converts dec to hex). This would allow us to see the decimal value still. Same goes for all of the other tests that do this.

Reply:
Good point, I implemented this.